### PR TITLE
Fix/nest example app test coverage

### LINKED ISSUE #303: Fix Nest Example App code coverage

### PR DESCRIPTION
**Pull request by IvayloGrahlyov**
resolves #303

### CODE REVIEW

**Review by PetarParushevMM — COMMENTED** [review]
I tested this and found a couple of things:
- For Nest Example App with Auth0 and Mongodb we are still not hitting our coverage on both unit and e2e tests (e2e coverage is quite far behind).
- Noticed that while our tests are still passing some of them are not quite accurate so it would be good to do some refactoring on them which can possibly done with another PR. Up for discussion.

Other than that I tested all other scenarios and they are all good with the coverage.

**Review by Yavorss — APPROVED** [review]

File: assets/nest/multiple-choice-features/users/mongodb/auth0/repositories/users.repository.spec.ts [hunk 1]
```diff
@@ -92,15 +92,13 @@ describe('UsersRepository', () => {
 
     const result = await usersRepository.updateOne(updatedUser._id, {
       email: updatedUser.email,
-      password: updatedUser.password,
     });
 
     expect(result).toBe(updatedUser);
     expect(findOneAndUpdate).toHaveBeenCalledWith(
-      { _id: updatedUser._id },
```
Comment (IvayloGrahlyov) [inline assets/nest/multiple-choice-features/users/mongodb/auth0/repositories/users.repository.spec.ts]: Shouldn't this be `_id` as it's how we store the users?

File: assets/nest/multiple-choice-features/authorization/mongodb/test/auth0/jest-e2e.config.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    '<rootDir>/jest.setup.ts'
+  ],
+  // path aliases from tsconfig.json
+  moduleNameMapper: {
+    '^@api/(.*)$': '<rootDir>/src/api/$1',
+    '^@database$': '<rootDir>/src/database',
+    '^@database/(.*)$': '<rootDir>/src/database/$1',
+    '^@extensions/(.*)$': '<rootDir>/src/extensions/$1',
+    '^@middleware$': '<rootDir>/src/middleware',
+    '^@utils/(.*)$': '<rootDir>/src/utils/$1',
```
Comment (PetarParushevMM) [inline assets/nest/multiple-choice-features/authorization/mongodb/test/auth0/jest-e2e.config.js:49]: Missing `'^@test/(.*)$': '<rootDir>/test/$1',` path. Causes e2e tests to fail

File: assets/nest/multiple-choice-features/authorization/mongodb/test/auth0/jest-e2e.config.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  coverageThreshold: {
+    global: {
+      branches: 85,
+      functions: 85,
+      lines: 85,
+      statements: 85
+    }
+  },
+  setupFiles: [
+    '<rootDir>/jest.setup.ts'
```
Comment (PetarParushevMM) [inline assets/nest/multiple-choice-features/authorization/mongodb/test/auth0/jest-e2e.config.js:40]: `jest.setup.ts` is not generated for mongodb example app. Tests are running even without the configuration so we should probably consider removing `setupFiles` prop from e2e config on mongodb.

**Review by PetarParushevMM — APPROVED** [review]

**Review by PetarParushevMM — APPROVED** [review]

**Pull request merged**